### PR TITLE
The piece_of_cake exercise is done.

### PR DESCRIPTION
**Pull request by leeKol**
Hi, I'd love for your review.

### CODE REVIEW

File: piece_of_cake.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    The function gets the ingredients needed to make a particular recipe, their prices per 100 grams,
+    and the quantities needed, and returns the price to be paid for buying the ingredients.
+    The function can get a list of ingredients that should be ignored when calculating.
+    :param prices: A dictionary of ingredients needed to make a particular recipe- the dictionary key will be the name
+    of the product, and the value of the dictionary will be its price per 100 grams.
+    :param optionals: A list of ingredients to be ignored while calculating the price.
+    :param quantities: For each ingredient- the amount in grams that we want to buy for the recipe.
+    :return: The price to pay for buying all the ingredients.
+    """
+    price = 0
```
Comment (itayday) [inline piece_of_cake.py]: I think that you should rename the variable price (it does not indicate that its the total price).

File: piece_of_cake.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+def get_recipe_price(prices, optionals=[], **quantities):
+    """
+    The function gets the ingredients needed to make a particular recipe, their prices per 100 grams,
+    and the quantities needed, and returns the price to be paid for buying the ingredients.
+    The function can get a list of ingredients that should be ignored when calculating.
```
Comment (talbarfm) [inline piece_of_cake.py:7]: Nice documentation!!!

File: piece_of_cake.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    :param prices: A dictionary of ingredients needed to make a particular recipe- the dictionary key will be the name
+    of the product, and the value of the dictionary will be its price per 100 grams.
+    :param optionals: A list of ingredients to be ignored while calculating the price.
+    :param quantities: For each ingredient- the amount in grams that we want to buy for the recipe.
+    :return: The price to pay for buying all the ingredients.
+    """
+    price = 0
+    for key in prices:
```
Comment (talbarfm) [inline piece_of_cake.py]: You can use reduce for this, and reduce this to one line

File: piece_of_cake.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+        if key not in optionals:
+            price += prices.get(key) * int(quantities.get(key)/100)
```
Comment (talbarfm) [inline piece_of_cake.py]: spaces aound operators

**Review by talbarfm — CHANGES_REQUESTED** [review]
Good job!!!

**Review by lovodkin93 — APPROVED** [review]
good job!

**Pull request merged**